### PR TITLE
Modified Task_DEV_2

### PR DESCRIPTION
**Pull request by PolinaYeroshevich**
Finding min and max numbers using methods.

### CODE REVIEW

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 1]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs:16]: Need to add blank line between methods.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 2]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
+                if (inputNumbers[i] > max)
+                {
+                    max = inputNumbers[i];
+                    indexMax = i;
+                }
+                Console.WriteLine(inputNumbers[i]);
+            }
+            inputNumbers[indexMin] = max;
+            inputNumbers[indexMax] = min;
+        }
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs:42]: Need to add blank line between methods.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 3]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
+        static void OutputArray(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine(inputNumbers[i]);
+            }
+        }
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs:50]: Need to add blank line between methods.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 4]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void ChangeMinMaxElement(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Would be better to name this method `SwapMinMaxElements`.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 5]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
+                {
+                    min = inputNumbers[i];
+                    indexMin = i;
+                }
+                if (inputNumbers[i] > max)
+                {
+                    max = inputNumbers[i];
+                    indexMax = i;
+                }
+                Console.WriteLine(inputNumbers[i]);
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Single responsibility violation.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 6]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
+        }
         static void Main(string[] args)
         {
-            Console.WriteLine("Enter the size of the array :");
+            Console.WriteLine("Enter the length of the array :");
             int arrayLength = Convert.ToInt32(Console.ReadLine());
 
             if (arrayLength <= 20)
             {
                 int[] inputNumbers = new int[arrayLength];
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Unnecessary blank line.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 7]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                int min = inputNumbers[0];
-                int max = inputNumbers[0];
-                int indexMin = 0;
-                int indexMax = 0;
+                InputArrayElement(inputNumbers, arrayLength);
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Unnecessary blank line.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 8]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
-                    if (inputNumbers[i] > max)
-                    {
-                        max = inputNumbers[i];
-                        indexMax = i;
-                    }
-                    Console.WriteLine(inputNumbers[i]);
-                }
-                inputNumbers[indexMin] = max;
-                inputNumbers[indexMax] = min;
+                ChangeMinMaxElement(inputNumbers, arrayLength);
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Unnecessary blank line.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 9]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
...
+                {
+                    max = inputNumbers[i];
+                    indexMax = i;
+                }
+                Console.WriteLine(inputNumbers[i]);
+            }
+            inputNumbers[indexMin] = max;
+            inputNumbers[indexMax] = min;
+        }
+        static void OutputArray(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Would be better to name it `PrintNumberArraryToConsole`. Or smth like this.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 10]
```diff
@@ -4,46 +4,59 @@ namespace Task_DEV_2
 {
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Please do smth like 
`static int[] CreateNumbersArray(int arrayLength)`

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 4 above)
Comment (VadimBelkin) [reply Task_DEV_2/Task_DEV_2/Program.cs]: This method should return changes array.

**Review by VadimBelkin — CHANGES_REQUESTED** [review]

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 1 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs:16]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 2 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs:42]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 3 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs:50]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 4 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 5 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 6 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 8 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 9 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 10 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 11]
```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
...
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                int min = inputNumbers[0];
-                int max = inputNumbers[0];
-                int indexMin = 0;
-                int indexMax = 0;
+        static int SwapMinMaxElements(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Why does this method return min element? This method has to return `inputNumbers`.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 12]
```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
...
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 20)
+            {
+                int[] inputNumbers = CreateNumbersArray(arrayLength);
+                PrintNumberArraryToConsole(inputNumbers, arrayLength);
+                SwapMinMaxElements(inputNumbers, arrayLength);
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: After swapping would be more logical do smth like this
`inputNumbers = SwapMinMaxElements(inputNumbers, arrayLength);`

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 13]
```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
...
+
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine(inputNumbers[i]);
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Why do you write enter size but variable is named `length`?

**Review by VadimBelkin — CHANGES_REQUESTED** [review]

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 11 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 12 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 13 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 14]
```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
...
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                int min = inputNumbers[0];
-                int max = inputNumbers[0];
-                int indexMin = 0;
-                int indexMax = 0;
+        static int [] SwapMinMaxElements(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Extra whitespace.

File: Task_DEV_2/Task_DEV_2/Program.cs [hunk 15]
```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
...
             }
+            inputNumbers[indexMin] = max;
+            inputNumbers[indexMax] = min;
+
+            return inputNumbers;
+        }
+
+        static void PrintNumberArraryToConsole(int[] inputNumbers, int arrayLength)
+        {
+
```
Comment (VadimBelkin) [inline Task_DEV_2/Task_DEV_2/Program.cs]: Extra empty line.

**Review by VadimBelkin — CHANGES_REQUESTED** [review]

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 14 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

File: Task_DEV_2/Task_DEV_2/Program.cs (hunk 15 above)
Comment (PolinaYeroshevich) [reply Task_DEV_2/Task_DEV_2/Program.cs]: Done.

**Review by VadimBelkin — APPROVED** [review]
Now it looks nice. Wait for approve from Maria and can merge.

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**